src/test/kotlin/com/forgerock/securebanking/framework/utils/AdminAuthentication.kt

```diff
@@ -0,0 +1,98 @@
+package com.forgerock.securebanking.framework.utils
+
+import com.forgerock.securebanking.framework.constants.IAM
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.isStatusRedirection
+import com.github.kittinunf.fuel.core.isSuccessful
+import com.google.gson.Gson
+
+//Identity Cloud Admin Credentials
+const val ADMIN_USERNAME = "amadmin"
+const val ADMIN_PASSWORD = "M4ymPctTY8wBG7lobdvmSTHc"
```

Comment:
Could we move this values to a property or environment variable to change them more easily? (as a DOMAIN env) because at the moment we need to update these values every day.

Reply:
I have created environment variables for these values.